benchmarks/addressable-equality/Gemfile.lock

```diff
@@ -0,0 +1,16 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.8.7)
+      public_suffix (>= 2.0.2, < 7.0)
+    public_suffix (6.0.2)
+
+PLATFORMS
+  aarch64-linux
```

Comment:
I guess that's more of a Bundler thing/issue, but I wonder why the architecture is even included here given those are pure-Ruby gems.

Reply:
AFAIK, that is the default behavior of bundler. It always include the architecture where the gemfile was created